README.md

```diff
@@ -51,3 +51,25 @@ so that you can use the OpenAI API SDKs with keyless (Entra) authentication. By
     ```
 
     This will use the OpenAI API SDK to make a request to the OpenAI API and print the response.
+
+
+## Creating a new azd enviroment 
```

Comment:
```suggestion
## Creating a new Azure OpenAI deployment
```

Yours is technically correct, but I'm going to reword it based on what I think the developer will be looking for.